libs/designsystem/item/src/item.component.spec.ts

```diff
@@ -19,243 +19,274 @@ describe('ItemComponent', () => {
-      spectator.component.disabled = true;
-      spectator.detectChanges();
+        expect(spectator.element).not.toHaveComputedStyle({ 'pointer-events': 'none' });
+      });
 
-      expect(spectator.element).toHaveComputedStyle({ 'pointer-events': 'none' });
-    });
+      it('should not have computed style "pointer:none" when not having "[disabled]" input defined', () => {
+        spectator.component.disabled = undefined;
+        spectator.detectChanges();
 
-    it(`should not have computed style "pointer:none" when having '[disabled]="false"`, () => {
-      spectator.component.disabled = false;
-      spectator.detectChanges();
+        expect(spectator.element).not.toHaveComputedStyle({ 'pointer-events': 'none' });
+      });
+
+      it('should have computed style "pointer:none" when having disabled attribute', () => {
+        spectator.element.setAttribute('disabled', '');
+        spectator.detectChanges();
 
-      expect(spectator.element).not.toHaveComputedStyle({ 'pointer-events': 'none' });
+        expect(spectator.element).toHaveComputedStyle({ 'pointer-events': 'none' });
+      });
     });
 
-    it('should not have computed style "pointer:none" when not having "[disabled]" input defined', () => {
-      spectator.component.disabled = undefined;
-      spectator.detectChanges();
+    describe('when configured with selectable', () => {
+      it('should not render a native button when selectable="false"', async () => {
+        spectator.setHostInput('selectable', false);
+        const ionItem = spectator.queryHost('ion-item');
+        await TestHelper.whenReady(ionItem);
 
-      expect(spectator.element).not.toHaveComputedStyle({ 'pointer-events': 'none' });
-    });
+        const nativePart = ionItem.shadowRoot.querySelector('[part="native"]');
+        expect(nativePart.tagName).not.toEqual('BUTTON');
+      });
 
-    it('should have computed style "pointer:none" when having disabled attribute', () => {
-      spectator.element.setAttribute('disabled', '');
-      spectator.detectChanges();
+      it('should render a native button when selectable="true"', async () => {
+        spectator.setHostInput('selectable', true);
+        const ionItem = spectator.queryHost('ion-item');
+        await TestHelper.whenReady(ionItem);
 
-      expect(spectator.element).toHaveComputedStyle({ 'pointer-events': 'none' });
+        const nativePart = ionItem.shadowRoot.querySelector('[part="native"]');
+        expect(nativePart.tagName).toEqual('BUTTON');
+      });
     });
   });
 
-  describe('when configured with selectable', () => {
-    it('should not render a native button when selectable="false"', async () => {
-      spectator.setHostInput('selectable', false);
+  describe('when configured with link', () => {
+    beforeEach(() => {
+      spectator = createHost(
+        '<kirby-item href="www.kirby.com" target="_blank" rel="noopener noreferrer" download>Value</kirby-item>'
```

Comment:
Does not really matter, and is arguably more boring, but consider linking here rather than an american vacuum cleaner company 😂
```suggestion
        '<kirby-item href="www.kirby.design" target="_blank" rel="noopener noreferrer" download>Value</kirby-item>'
```